tests/positive/filesystems/reformat_filesystem.go

```diff
@@ -49,7 +49,7 @@ func ReformatToBTRFS_2_0_0() types.Test {
 	        "format": "btrfs",
 	        "create": {
 	          "force": true,
-	          "options": [ "--label=OEM", "--uuid=CA7D7CCB-63ED-4C53-861C-1742536059D4" ]
+	          "options": [ "--label=OEM", "--uuid=$uuid12" ]
```

Comment:
since each test has its own UUID pool, we should keep the N in `$uuidN` as low as possible so it's easy to keep track of it.